default/scripting/buildings/colonies/SP_ABADDONI.focs.txt

```diff
@@ -64,7 +64,7 @@ BuildingType
     ]
     effectsgroups = [
         [[LIFECYCLE_MANIP_POPULATION_EFFECTS("SP_ABADDONI")]]
-        
```

Comment:
This is not python script.

Reply:
This was generated by a python script and the PEP-8 conformance directly impacts that generated output.